actions/release-it/.release-it.js

```diff
@@ -0,0 +1,54 @@
+const version = "${version}";
+const packageName = process.env.npm_package_name;
+const monorepo = process.env.monorepo;
+const branch_name = process.env.branch_name;
+const scope = packageName.split("/")[1];
+
+const releaseBranches = ["main", "master", "release/**"];
+let increment = undefined;
+
+// Check if branch conditions are met
+if (monorepo) {
+  if (branch_name.indexOf("release") > -1 && branch_name.indexOf(scope) < 0) {
+    console.log(`Skipping release of ${scope} since it should not be released from branch ${branch_name}`);
+    process.exit(0);
+  }
+}
```

Comment:
Check for monorepos. Maybe it is better to create our own plugin for these things instead. I was not able to load the plugin properly, but that should be possible to fix